tests/unit/select_connection_ioloop_tests.py

```diff
@@ -35,34 +46,43 @@ def setUp(self):
         self.ioloop = select_connection.IOLoop()
 
     def tearDown(self):
-        self.ioloop.remove_timeout(self.fail_timer)
         self.ioloop = None
 
     def start(self):
-        self.fail_timer = self.ioloop.add_timeout(self.TIMEOUT, self.on_timeout)
+        ''' Setup timeout handler for detecting 'no-activity'
+            and start polling. '''
+        fail_timer = self.ioloop.add_timeout(self.TIMEOUT, self.on_timeout)
+        self.addCleanup(self.ioloop.remove_timeout, fail_timer)
         self.ioloop.start()
 
     def on_timeout(self):
         """called when stuck waiting for connection to close"""
         # force the ioloop to stop
         self.ioloop.stop()
-        raise AssertionError('Test timed out')
+        self.fail('Test timed out')
 
 
 class IOLoopThreadStopTestSelect(IOLoopBaseTest):
+    ''' Test ioloop being stopped by another Thread. '''
     SELECT_POLLER = 'select'
     def start_test(self):
-        t = threading.Timer(0.1, self.ioloop.stop)
-        t.start()
+        ''' Starts a thread that stops ioloop after a while
+            and start polling '''
+        # Pylint wants ids at least 3chars long. That's the reason for 't__'
+        t__ = threading.Timer(0.1, self.ioloop.stop)
```

Comment:
@shinji-s: `t__` is not a good idea; it's not conventional. If you really needed a longer variable, it would have been better to change it to `timer` in this case. However, since the variable in this case is only used to start the timer, you could get rid of it altogether: `threading.Timer(0.1, self.ioloop.stop).start()` is all you need here